docs/rules/no-extra-parens.md

```diff
@@ -34,8 +36,6 @@ The following patterns are not considered problems:
 
 ({}.toString.call());
 
-(function(){} ? a() : b())
```

Comment:
The better change would be to move the closing parenthesis after the function. This example was meant to illustrate the fact that some values may be parenthesised for reasons other than precedence.